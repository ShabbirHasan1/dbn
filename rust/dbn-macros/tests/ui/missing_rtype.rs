use dbn_macros::dbn_record;

#[repr(C)]
#[dbn_record]
struct Record {
    pub a: u8,
}

fn main() {}
