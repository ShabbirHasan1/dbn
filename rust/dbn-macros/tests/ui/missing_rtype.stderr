error: Need to specify at least one rtype to match against
 --> tests/ui/missing_rtype.rs:4:1
  |
4 | #[dbn_record]
  | ^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `dbn_record` (in Nightly builds, run with -Z macro-backtrace for more info)
